### PR TITLE
Add support for Swift 2.3

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: Sources/SectionScrubber.swift [hunk 1]
```diff
@@ -224,7 +224,11 @@ public class SectionScrubber: UIView {
             return indexPath
         }
         for indexPath in collectionView.indexPathsForVisibleSupplementaryElementsOfKind(UICollectionElementKindSectionHeader) {
-            let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)
+            #if swift(>=2.3)
+                let view = collectionView.supplementaryViewForElementKind(UICollectionElementKindSectionHeader, atIndexPath: indexPath)!
```
Comment (3lvis) [inline Sources/SectionScrubber.swift]: :dolan:

Comment (marijnschilling) [reply Sources/SectionScrubber.swift]: Is it safe to force unwrap it?

Comment (3lvis) [reply Sources/SectionScrubber.swift]: Maybe not, probably not an issue for now. What do you think?

Comment (3lvis) [reply Sources/SectionScrubber.swift]: https://github.com/bakkenbaeck/SectionScrubber/issues/40

Comment (marijnschilling) [reply Sources/SectionScrubber.swift]: We could make the view an optional above the #if clouse. Fill it with optional result in >=swift 2.3, fill it with view in swift lower than that. The function returns an optional indexpath anyways :v:

Comment (3lvis) [reply Sources/SectionScrubber.swift]: The Swift 2.2 code will be removed once Xcode 8 is released, I think it's better to not do much refactoring to have 2.3 support.

Comment (marijnschilling) [reply Sources/SectionScrubber.swift]: well if it will be an optional by default in the future we'll need to deal with the optional even more right?

Comment (3lvis) [reply Sources/SectionScrubber.swift]: You're right. I'll see what I can do :)

**Comment by 3lvis** [comment]
@marijnschilling what about now, better? :)

**Comment by marijnschilling** [comment]
Looks good! So when the view isnt found a nil indexpath is returned, thats perfect!

**Comment by 3lvis** [comment]
Oh, duh, you were supposed to merge it.

So much :dolan:

**Comment by marijnschilling** [comment]
its fine!

**Pull request merged**